client/tooltool.py

```diff
@@ -695,8 +685,12 @@ def add_files(manifest_file, algorithm, filenames, version, visibility, unpack):
     for old_fr in old_manifest.file_records:
         if old_fr.filename not in new_filenames:
             new_manifest.file_records.append(old_fr)
-    with open(manifest_file, **open_attrs) as output:
-        new_manifest.dump(output, fmt='json')
+    if PY3:
+        with open(manifest_file, mode="w") as output:
+            new_manifest.dump(output, fmt='json')
+    else:
+        with open(manifest_file, mode="wb") as output:
+            new_manifest.dump(output, fmt='json')
```

Comment:
Not sure if we can use "wb" regardless of the python version, but it should work fine for a text file in any case.

Reply:
I actually originally had this as just "w" for both versions (it's just a text file after all), but it broke some tests (https://github.com/mozilla-releng/tooltool/pull/27/commits/fad09c9f84f281d60645a3e0ebde111ce6b5a1d5).

I think `wb` isn't actually necessary for python2, except for the tests.